hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -1819,6 +1820,19 @@ void removeBlocksAssociatedTo(final DatanodeStorageInfo storageInfo) {
         storageInfo, node);
   }
 
+  /** Remove the blocks to the given DatanodeDescriptor from InvalidateBlocks. */
+  void removeBlocksFromInvalidateBlocks(final DatanodeDescriptor node) {
```

Comment:
Suggest modifying the method name: 
removeBlocksFromInvalidateBlocks -> removeNodeFromInvalidateBlocks